microsoft/atlas-design
# Radio

Task: task-[531899](https://dev.azure.com/ceapex/Engineering/_workitems/edit/531899/)

Link: preview-[358](https://design.docs.microsoft.com/pulls/358)

This PR adds a radio component.

Note: radio with images is not being included. It has been depreciated from assessment. Report Feedback still uses it, but verified with PMs that the images can be removed as the pattern is not good for accessibility.

## Testing

1. Visit [the radio page](https://design.docs.microsoft.com/pulls/358/components/radio.html).
2. Check all themes and screensize.
